Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+fhBuildNode() {
+    stage('Install Dependencies') {
+        npmInstall {}
+    }
+
+    stage('Unit Tests') {
+            sh 'grunt fh:unit'
+    }
+
+    stage('Integration Tests') {
+        print "Integration tests are disabled due to https://issues.jboss.org/browse/RHMAP-18033"
```

Comment:
This is not a private internal project, so the Jira link should be publicly accessible in the FH project. Could you move that please, and make any necessary adjustments?